security.md

```diff
@@ -73,6 +74,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2018-1096"></a>CVE-2018-1096: SQL injection on dashboard 
+
+One of the parameters passed when saving dashboard widget positions was not properly escaped, leading to possibility of SQL injection attack. Due to the nature of the query the impact is limited to possible information disclosure and does not allow modifications to the database.
+
+This issue was reported by Martin Povolný of Red Hat.
+
+* Affects Foreman 1.9 and higher.
+* Fix released with Foreman 1.16.1.
```

Comment:
I guess we'll have to wait until 1.16.1 is out or change the string to "will be fixed" and then send a separate commit.

Reply:
1.16.1 is being released right now, i guess it's safe to merge this now